ml-algorithms/src/main/java/org/opensearch/ml/engine/algorithms/agent/AgentUtils.java

```diff
@@ -946,4 +954,45 @@ public static void cleanUpResource(Map<String, Tool> tools) {
             }
         }
     }
+
+    /**
+     * Generates a formatted current date and time string in UTC timezone.
+     * 
+     * <p>This method returns the current date and time formatted according to the provided pattern.
+     * If no format is provided or the format is invalid, it uses the default format:
+     * "yyyy-MM-dd'T'HH:mm:ss'Z'" (e.g., "2024-01-15T14:30:00Z").
```

Comment:
Suggestion from AI
 While "Z" at the end of the ISO 8601 timestamp does indicate UTC (Zulu time), it's not universally understood by all readers. For better clarity, especially when communicating with non-technical audiences, you might want to either:

    Explicitly add "UTC": 2024-01-15T14:30:00 UTC
    Or keep the Z but add "(UTC)": 2024-01-15T14:30:00Z (UTC)

Reply:
I'm good with this change.